Builder failure in github gradle workflow

### Description

gradle build log

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':importer:compileTestJava'.
> Could not resolve all files for configuration ':importer:testCompileClasspath'.
   > Could not find org.apache.jclouds:jclouds-allblobstore:2.6.0-SNAPSHOT.
     Required by:
         project :importer > org.gaul:s3proxy:2.1.0-SNAPSHOT:20230816.115951-3
   > Could not find org.apache.jclouds.api:filesystem:2.6.0-SNAPSHOT.
     Required by:
         project :importer > org.gaul:s3proxy:2.1.0-SNAPSHOT:20230816.115951-3
   > Could not find org.apache.jclouds.driver:jclouds-slf4j:2.6.0-SNAPSHOT.
     Required by:
         project :importer > org.gaul:s3proxy:2.1.0-SNAPSHOT:20230816.115951-3

```

### Steps to reproduce

Run `./gradlew build`

### Additional context

_No response_

### Hedera network

other

### Version

v0.87.0-SNAPSHOT

### Operating system

None

--- a/buildSrc/src/main/kotlin/java-conventions.gradle.kts
+++ b/buildSrc/src/main/kotlin/java-conventions.gradle.kts
@@ -41,6 +41,9 @@ repositories {
     maven {
         url = uri("https://hyperledger.jfrog.io/artifactory/besu-maven/")
     }
+    maven {
+        url = uri("https://repository.apache.org/content/repositories/snapshots")
+    }
 }
 
 dependencyManagement {

--- a/buildSrc/src/main/kotlin/java-conventions.gradle.kts
+++ b/buildSrc/src/main/kotlin/java-conventions.gradle.kts
@@ -42,6 +42,8 @@ repositories {
         url = uri("https://hyperledger.jfrog.io/artifactory/besu-maven/")
     }
     maven {
+        // org.gaul:s3proxy:2.1.0-SNAPSHOT depends on snapshot artifacts from the repo, remove it when upgrading
+        // s3proxy to a GA release
         url = uri("https://repository.apache.org/content/repositories/snapshots")
     }
 }

